styles/style.css

```diff
@@ -0,0 +1,1147 @@
+} 
+
+.topic__heading {
+    grid-column: 2;
+    align-self: start;
+    color: #0a3d61;
+    font-size: 1.45rem;
+    margin: 0 0 1rem 0;
+}
+
+.topic__heading::first-letter {
+    text-transform: uppercase;
+}
+
+.grid-article > p {
+    font-size: inherit;
+    line-height: 2rem;
+    margin: 0;
+}
+
+.blog-article__text {
+    font-size: inherit;
+    line-height: 2rem;
+    margin: 0;
+}
+
+/* ---- Dog's Page ---- */
+.dogs {
+    display: flex;
+    flex-direction: column;
+}
+
+.dog-heading {
+    margin: 1.5rem 0 0 13%;
+    /* margin: 0; */
+    padding: 0;
+}
+
+/* ---- Dog's Page - Main Section ---- */
+.bio_puppies {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 1rem;
+    margin: 1.5rem 13% 3rem 13%;
+}
+
+/* .bio_puppies > article  */
+.dog-card {
+    border:rgb(224, 224, 226);
+    border-style: solid;
+    border-top-color:#fff;
+    border-width: 2px;
+}
+
+/* ---- All pages ---- */
+h3 {
+    display: none;
+}
+/* ---- Homepage and Dog's Page - Cards (dog's bio) ---- */
+
+/* .dog-card  */
+h2 {
+    color: #0a3d61;
+    font-size: 1.3rem;
+    margin: .2rem .8rem;
+}
+
+/* .dog-card >  */
+h2::first-letter {
+    text-transform: uppercase;
+}
+
+.card {
+    max-width: 100%; 
+    width: 100%;
+}
+
+/* ---- Hover effects for image cards ---- */
+.card:hover {
+    opacity: 1;
+    -webkit-animation: flash 1.5s;
+    animation: flash 1.5s;
+}
+
+@-webkit-keyframes flash {
+    0% {
+        opacity: .4;
+    }
+    100% {
+        opacity: 1;
+    }
+}
+@keyframes flash {
+    0% {
+        opacity: .4;
+    }
+    100% {
+        opacity: 1;
+    }
+}
+
+span.cost {
+    font-weight: bold;
+}
+
+small.dog-name {
+    color: #000;
+    font-size: .8rem;
+    margin: .8rem;
+    padding: 0;
+}
+
+.dog-name {
+    font-size: .94rem;
+    margin: .8rem;  
+    padding: 0;  
+}
+
+input[value="Adopt"] {
+    display: block;
+    background-color: #a1c349;
+    color: #fff;
+    font-weight: bold;
+    margin-right: auto;
+    margin-bottom: 1rem;
+    margin-left: auto;
+    padding: 0.5rem 4rem;
+    border: none;
+    border-radius: 2px;
+    transition: border .4s ease, color .4s ease, background-color .4s ease;
+}
+
+input[value="Adopt"]:hover {
+    background-color: #fff;
+    color: #a1c349;
+
+}
+
+
+/* ---- Footer ---- */
+footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: #0a3d61;
+    font-size: 1rem;
+}
+
+footer img {
+    width: 75%;
+    padding-top: 3rem;
+}
+
+footer > p {
+    color: #ffffff;
+    text-align: center;
+    line-height: 1.8rem;
+    margin: 0 32%;
+}
+
+.bottom-nav {
+    margin: 1.5rem 0 1.5rem 0;
+}
+
+ul.bottom-nav a {
+    color: #fff;
+}
+
+ul.bottom-nav > li:after {
+    color: #fff;
+}
+
+a:first-child {
+    padding-left: 0;
+}
+
+small {
+    color: #fff;
+    margin-top: 1rem;
+    padding-bottom: 4rem;
+}
+
+
+/* MEDIA QUERIES */
+@media screen and (min-width: 0px) and (max-width: 576px) {

```

Comment:
Great work with the media queries. Note you do not need to specify a min-width here. If you just set a max-width, the compiler will allow for anything under your max-width.

Reply:
Thanks Tyler. I was honestly not sure enough to define the 'minimum width' with the value of 0. I thought this idea will keep the layout strictly accurate to the specified style with the root value of 0. So some best practices recommend 'minimum widths' rather than 'maximum widths', but it makes sense here. Thanks again for your help, sometimes simpler is better.